### PR TITLE
Functions - Tasks and solutions

### PR DESCRIPTION
**Pull request by Dheyson**
Translate tasks and solutions

- if-else-required
- rewrite-function-question-or
- min
- pow

### CODE REVIEW

File: 1-js/02-first-steps/14-function-basics/article.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ showMessage();
...
-The call `showMessage()` executes the code of the function. Here we will see the message two times.
+A chamada `showMessage()` executa o código da função. Aqui vemos a mensagem duas vezes.
 
-This example clearly demonstrates one of the main purposes of functions: to avoid code duplication.
+Este exemplo demonstra claramente um dos principais objetivos das funções: evitar código duplicado.
 
-If we ever need to change the message or the way it is shown, it's enough to modify the code in one place: the function which outputs it.
+Se precisarmos mudar a mensagem ou a maneira que ela é mostrada, basta modificar o código em um só lugar: a função que gera isso.
 
 ## Local variables
```
Comment (luizclr) [inline 1-js/02-first-steps/14-function-basics/article.md]: could you translate this session?

Comment (Dheyson) [reply 1-js/02-first-steps/14-function-basics/article.md]: Sure, sorry. I think I just forget that one.

Comment (Dheyson) [reply 1-js/02-first-steps/14-function-basics/article.md]: /done

Comment (Dheyson) [reply 1-js/02-first-steps/14-function-basics/article.md]: @luizclr I am sorry, but for me the paragraph it already translated, is there something about the translation?
I am missing something. I am not doing right with the bot also, it says to put done. I did right down below, but it seems I did something wrong.

Comment (Dheyson) [reply 1-js/02-first-steps/14-function-basics/article.md]: Translated the header 2 title from the section Local variables. was it only this to check?, or wanna to verify more the last paragraph translation?
I translated, but it seems the next paragraph for the section Local variables appear to be translated. I don't know if you agree, but if everything is okay, we could end this PR and I translate the section, which I am right now reviewing.
Thank you

**Pull request merged**